test/test_limits.py

```diff
@@ -21,6 +21,22 @@ async def test_max_client_conn(bouncer):
     await result_last
 
 
+@pytest.mark.asyncio
+async def test_max_db_client_connections(bouncer):
+    test_db = "conn_limit_db"
+    test_user = "muser1"
+    result = bouncer.asleep(3, dbname=test_db, user=test_user)
+    await asyncio.sleep(1)
+    # should still be allowed, since it's the last allowed connection
+    await bouncer.atest(dbname=test_db, user=test_user)
+    result_last = bouncer.asleep(3, dbname=test_db, user=test_user)
+    await asyncio.sleep(1)
+    with pytest.raises(psycopg.OperationalError, match=r"max_db_client_connections"):
+        await bouncer.atest(dbname=test_db, user=test_user)
```

Comment:
if it's easy, could you also check the values of `max_client_connections` and `current_client_connections` using `SHOW DATABASES`? Coverage of our stats commands is currentl very minimal (or non-existent). If it's not too hard to add, this would be a good moment to improve that.

Reply:
Will look into this.

Reply:
This should be good now.